nixos/modules/services/networking/nghttpx/nghttpx-options.nix

```diff
@@ -132,7 +132,7 @@
       type        = lib.types.int;
       default     = 0;
       description = ''
-        Set maximum number of open files (RLIMIT_NOFILE) to <N>. If 0
```

Comment:
@Mic92 how did you check this so I can know to fix this sort of thing in the future?

Reply:
I noticed it, when adding the service to my `configuration.nix` because this triggers a manual rebuild.
I can be also build using: https://nixos.wiki/wiki/Contributing_to_Nix_documentation#How_to_build_the_NixOS_manual